pkg/proxy/nftables/proxier_test.go

```diff
@@ -103,6 +103,12 @@ func NewFakeProxier(ipFamily v1.IPFamily) (*knftables.Fake, *Proxier) {
 
 	nft := knftables.NewFake(nftablesFamily, kubeProxyTable)
 
+	var nodeIP net.IP
+	if ipFamily == v1.IPv4Protocol {
+		nodeIP = netutils.ParseIPSloppy(testNodeIP)
+	} else {
+		nodeIP = netutils.ParseIPSloppy(testNodeIPv6)
+	}
```

Comment:
ah, good catch... there aren't very many IPv6 tests so I guess we weren't doing anything where using the right nodeIP mattered yet